bitmaps.h

```diff
@@ -0,0 +1,211 @@
+
+		std::wstring name;
+		int resource_id;
+		BitmapGroup group;
+		BitmapData *data;
+	};
+
+	static BitmapData *load(int resource_id);
+
+	inline const static Definition Cvjoy = {
+		.name = L"cvjoy",
+		.resource_id = IDB_BITMAP1,
+		.group = BitmapGroup::Impostor,
+		.data = load(IDB_BITMAP1),
+	};
+
+	inline const static Definition Fn = {
+		.name = L"fn",
+		.resource_id = IDB_BITMAP2,
+		.group = BitmapGroup::Impostor,
+		.data = load(IDB_BITMAP2),
+	};
+
+	inline const static Definition Fnplead = {
+		.name = L"fnplead",
+		.resource_id = IDB_BITMAP3,
+		.group = BitmapGroup::Impostor,
+		.data = load(IDB_BITMAP3),
+	};
+
+	inline const static Definition Lk = {
+		.name = L"lk",
+		.resource_id = IDB_BITMAP4,
+		.group = BitmapGroup::Yokin,
+		.data = load(IDB_BITMAP4),
+	};
+
+	inline const static Definition Lkconcern = {
+		.name = L"lkconcern",
+		.resource_id = IDB_BITMAP5,
+		.group = BitmapGroup::Yokin,
+		.data = load(IDB_BITMAP5),
+	};
+
+	inline const static Definition Lkcool = {
+		.name = L"lkcool",
+		.resource_id = IDB_BITMAP6,
+		.group = BitmapGroup::Yokin,
+		.data = load(IDB_BITMAP6),
+	};
+
+	inline const static Definition Lkexhausted = {
+		.name = L"lkexhausted",
+		.resource_id = IDB_BITMAP7,
+		.group = BitmapGroup::Yokin,
+		.data = load(IDB_BITMAP7),
+	};
+
+	inline const static Definition Lkhusk = {
+		.name = L"lkhusk",
+		.resource_id = IDB_BITMAP8,
+		.group = BitmapGroup::Yokin,
+		.data = load(IDB_BITMAP8),
+	};
+
+	inline const static Definition Lkjoy = {
+		.name = L"lkjoy",
+		.resource_id = IDB_BITMAP9,
+		.group = BitmapGroup::Yokin,
+		.data = load(IDB_BITMAP9),
+	};
+
+	inline const static Definition Lkmoyai = {
+		.name = L"lkmoyai",
+		.resource_id = IDB_BITMAP10,
+		.group = BitmapGroup::Impostor,
+		.data = load(IDB_BITMAP10),
+	};
+
+	inline const static Definition Lksix = {
+		.name = L"lksix",
+		.resource_id = IDB_BITMAP11,
+		.group = BitmapGroup::Yokin,
+		.data = load(IDB_BITMAP11),
+	};
+
+	inline const static Definition Lkthink = {
+		.name = L"lkthink",
+		.resource_id = IDB_BITMAP12,
+		.group = BitmapGroup::Yokin,
+		.data = load(IDB_BITMAP12),
+	};
+
+	inline const static Definition Lkthumbsup = {
+		.name = L"lkthumbsup",
+		.resource_id = IDB_BITMAP13,
+		.group = BitmapGroup::Yokin,
+		.data = load(IDB_BITMAP13),
+	};
+
+	inline const static Definition Lkunamused = {
+		.name = L"lkunamumsed",
+		.resource_id = IDB_BITMAP14,
+		.group = BitmapGroup::Yokin,
+		.data = load(IDB_BITMAP14),
+	};
+
+	inline const static Definition Lkxd = {
+		.name = L"lkxd",
+		.resource_id = IDB_BITMAP15,
+		.group = BitmapGroup::Yokin,
+		.data = load(IDB_BITMAP15),
+	};
+
+	inline const static Definition Nx = {
+		.name = L"nx",
+		.resource_id = IDB_BITMAP16,
+		.group = BitmapGroup::Impostor,
+		.data = load(IDB_BITMAP16),
+	};
+
+	inline const static Definition Vx = {
+		.name = L"vx",
+		.resource_id = IDB_BITMAP17,
+		.group = BitmapGroup::Impostor,
+		.data = load(IDB_BITMAP17),
```

Comment:
Another unfortunate bit of necessary duplication between `.resource_id` and `.data`.